src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -122,6 +122,7 @@ Copyright (c) .NET Foundation. All rights reserved.
                                        RuntimeConfigDevPath="$(ProjectRuntimeConfigDevFilePath)"
                                        RuntimeIdentifier="$(RuntimeIdentifier)"
                                        PlatformLibraryName="$(MicrosoftNETPlatformLibrary)"
+                                       RuntimeFrameworkVersion="$(RuntimeFrameworkVersion)"
```

Comment:
There's a bit of explanation of this in the comments on the property as it's defined in the task.  This allows the version of the shared framework written to the runtimeconfig.json file to be configured independently of the package version.  Since we don't update the target framework version for patch versions of the shared framework, this is the way you would choose a patch version of the runtime.  It would also allow us to update the Microsoft.NETCore.App package so that the latest version supportss targeting all previous versions of .NET Core, by putting TargetFrameworkVersion-specific `.targets` files in the NuGet package that would set this property to the appropriate shared framework version for each supported TargetFrameworkVersion.